.github/workflows/e2e_tests.yml

```diff
@@ -91,6 +91,7 @@ jobs:
         run: pnpm exec playwright test --project="${{ matrix.browser }}"
         env:
           URL: ${{ env.URL }}
+          VERCEL_PROTECTION_BYPASS_SECRET: ${{ secrets.VERCEL_PROTECTION_BYPASS_SECRET }}
```

Comment:
you can find this value in `https://vercel.com/*/*/settings/deployment-protection`

see [Protection Bypass for Automation](https://vercel.com/docs/deployment-protection/methods-to-bypass-deployment-protection/protection-bypass-automation)